src/main/kotlin/io/gitlab/arturbosch/detekt/idea/webviews/RulesBrowserService.kt

```diff
@@ -57,7 +57,7 @@ class RulesBrowserService : Disposable {
     }
 
     private fun updateStyles() {
-        val ideTheme = if (StartupUiUtil.isUnderDarcula()) "dark" else "light"
```

Comment:
hm, is it internal in future IJ versions than 222?

![Screenshot from 2024-03-31 10-11-40](https://github.com/detekt/detekt-intellij-plugin/assets/20924106/01f5faa9-74cd-4009-ad7d-b227d4527aef)

Reply:
Doesn't look like from the screenshot... But worst case scenario you can use `JBColor.isBright()`